src/Components/Notes/NoteCardCreate.jsx

```diff
@@ -0,0 +1,187 @@
+import {useAuth0} from '@auth0/auth0-react'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import CardContent from '@mui/material/CardContent'
+import CardHeader from '@mui/material/CardHeader'
+import InputBase from '@mui/material/InputBase'
+import Paper from '@mui/material/Paper'
+import useTheme from '@mui/styles/useTheme'
+import {TooltipIconButton} from '../Buttons'
+import useStore from '../../store/useStore'
+import {postIssue} from '../../utils/GitHub'
+import Submit from '../../assets/icons/Submit.svg'
+
+
+/**
+ * Note card create
+ *
+ * @param {string} username
+ * @param {string} avatarUrl
+ * @return {object} React component
+ */
+export default function NoteCardCreate({
+  username = '',
+  avatarUrl = '',
+}) {
+  const repository = useStore((state) => state.repository)
+  const toggleIsCreateNoteActive = useStore((state) => state.toggleIsCreateNoteActive)
+  const [title, setTitle] = useState('')
+  const [body, setBody] = useState('')
+  const {user, isAuthenticated} = useAuth0()
+  const accessToken = useStore((state) => state.accessToken)
+  const createdNotes = useStore((state) => state.createdNotes)
+  const setCreatedNotes = useStore((state) => state.setCreatedNotes)
+  const theme = useTheme()
+
+
+  /**
+   * create issue takes in the title and body of the note
+   *
+   */
+  function isCreateNoteActive() {
+    // TODO(Oleg) noteIndex is used for note number :: need to introduce internal index system
+    let noteIndex
+    if (createdNotes) {
+      noteIndex = Object.keys(createdNotes).length
+    } else {
+      noteIndex = 1
+    }
+    const issuePayload = {
+      title,
+      body,
+    }
+    postIssue(repository, issuePayload, accessToken)
+    toggleIsCreateNoteActive()
+    const localNote = {
+      index: noteIndex,
+      id: noteIndex,
+      number: noteIndex,
+      title: title,
+      body: body,
+      date: new Date().toISOString(),
+      username: user ? user.name : 'username',
+      avatarUrl: user ? user.picture : '',
+      numberOfComments: '',
+      synchedNote: false,
+    }
+    if (createdNotes === null) {
+      setCreatedNotes([localNote])
+    } else {
+      setCreatedNotes([localNote, ...createdNotes])
+    }
+  }
+
+  return (
+    <Paper
+      elevation={1}
+      variant='note'
+      square
+      sx={{
+        width: '100%',
+
+      }}
+    >
+      <CardHeader
+        title={
+          <InputField
+            placeholder={'Note Title'}
+            inputText={title}
+            setInputText={setTitle}
+            multiline={false}
+            maxLength={256}
+          />}
+        avatar={
+          isAuthenticated ?
+          <Avatar
+            alt={user.name}
+            src={user.picture}
+            sx={{width: 22, height: 22}}
+          /> :
+          <Avatar alt={username} src={avatarUrl}/>
+        } sx={{
+          backgroundColor: theme.palette.primary.main,
+        }}
+      />
+      <CardContent
```

Comment:
Per convo.. check to see if you can avoid doing custom styling with sx here, and the Box below.  I think theme has keys for these and will make UI consistent across components

Reply:
will address in a separate pr.

Reply:
will address in a separate pr.